editoast/src/core/simulation.rs

```diff
@@ -203,6 +203,19 @@ impl PhysicsConsistParameters {
             self.traction_engine.rolling_resistance.clone()
         }
     }
+
+    pub fn compute_gamma(&self) -> Gamma {
+        self.towed_rolling_stock
+            .as_ref()
+            .map(|towed| {
+                if towed.gamma.value < self.traction_engine.gamma.value {
+                    towed.gamma.clone()
+                } else {
+                    self.traction_engine.gamma.clone()
+                }
```

Comment:
Once https://github.com/OpenRailAssociation/osrd/pull/9887 will be merged, you might be able to express that as `min`.

```suggestion
                std::cmp::min(towed.const_gamma, self.traction_engine.const_gamma)
```

Reply:
Should we wait for this to merge this PR ?